tests/ui/modules/mod_file_disambig.rs

```diff
@@ -0,0 +1,5 @@
+//! related issue <https://github.com/rust-lang/rust/issues/4116>
+
+mod mod_file_disambig_aux; //~ ERROR file for module `mod_file_disambig_aux` found at both
+
+fn main() {}
```

Comment:
original

```rust
mod mod_file_disambig_aux; //~ ERROR file for module `mod_file_disambig_aux` found at both

fn main() {
    assert_eq!(mod_file_aux::bar(), 10);
    //~^ ERROR failed to resolve: use of unresolved module or unlinked crate `mod_file_aux`
}
```

https://github.com/rust-lang/rust/commit/a74d92e8ab61863876d2b5b6256b403efbb55492

The assert_eq statement here seems unnecessary, so I removed it.